msg_prios.py

```diff
@@ -0,0 +1,15 @@
+"""OpenFlow message priorities used in the core queues."""
+
+from pyof.v0x04.common.header import Type
+
+
+def of_msg_prio(msg_type: int) -> int:
+    """Get OpenFlow message priority.
+
+    The lower the number the higher the priority, if same priority, then it
+    will be ordered ascending by KytosEvent timestamp."""
+    prios = {
+        Type.OFPT_FLOW_MOD.value: 1000,
```

Comment:
I initially chose 1000 to leave enough room for other messages in between that will likely be added here in the future. OFPT_FLOW_MOD and OFPT_BARRIER_REQUEST, by default, whe using this function, will have a lower priority (higher number) than default messages since they can have a significant rate of messages.